consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/saker/mediation/SakApiGateway.kt

```diff
@@ -0,0 +1,89 @@
+package no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.saker.mediation
```

Comment:
`mediation` ?

Reply:
Ja , i Fred George sin kurs vi lært om dele koden i  logisk 3 deler(Man kan vurdere å lage 3 seperate moduler også )
1. API - Tjeneste lag 
2. Mediation : Kommunisering med ekstern systemer
3. Domain

Reply:
Er ikke etter pattern som er brukt tidligere i kodebasen, så derfor jeg stusset på at det ble introdusert. 
Men hvorvidt "SakRestClient" (som jeg sannsynligvis hadde kalt den :P ) blir kalt `SakApiGateway`  og ligger i `mediation` har ikke så mye å si for min del